### DISCUSSION TITLE
Using MPR121 capacitive touch devices

### DISCUSSION THREAD

**Discussion by ghost** [tttapa/Control-Surface#466] [Q&A]
Hi guys ! I was wondering if someone has already setup a controller using MPR121 capacitive touch sensors or if anyone knows what to do? I'm currently working on a project using 3 of these iCs and a few pots and I'm getting stuck with the code... I don't really get how I can get the MPR121 to work with the Control Surface library.. Any help or advice is more than welcome !

**Comment by tttapa** [comment]
For future reference: https://forum.arduino.cc/t/arduino-midi-controller-with-mpr121/859570/10
